sdk/identity/identity/src/util/tracingUtils.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should these helpers move to some core library?

Reply:
The `getSpanOptions` is not required. A call to this can be replaced with `options && options.spanOptions`

The `createSpan` can indeed move to `core-tracing`, but I would keep work out of this PR.